src/components/LeaderBoard.jsx

```diff
@@ -0,0 +1,249 @@
+import React, { Component, Fragment } from 'react';
+import Avatar from 'react-avatar';
+import PropTypes from 'prop-types';
+import moment from 'moment';
+import { Link } from 'react-router-dom';
+
+import config from 'configuration';
+import Loader from './Loader';
+import { getUserName, getUserAvatar, convertEthHelper } from '../lib/helpers';
+import Donation from '../models/Donation';
+
+/**
+ * Shows a table of aggregateDonations for a given type (dac, campaign, milestone)
+ */
+
+class LeaderBoardItem extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      showDetails: false,
+      itemType: null, // Delegated ro Directly donated
+      hasHistory: this.props.d.status === Donation.PAID, // PAID donations always has history
```

Comment:
No PAID donation is sent from backend... every where status is compared to not be PAID should be removed